lib/setupmanagers/hckclient.rb

```diff
@@ -76,10 +78,18 @@ def run_post_start_commands
       @project.engine.drivers&.each do |driver|
         driver.post_start_commands&.each do |command|
           desc = command.desc
-          cmd = command.guest_run
+          guest_run = command.guest_run
+          host_run = command.host_run
+
+          if guest_run
+            @logger.info("Running command (#{desc}) on client #{@name}")
+            @tools.run_on_machine(@name, desc, guest_run)
+          end
 
-          @logger.info("Running command (#{desc}) on client #{@name}")
-          @tools.run_on_machine(@name, desc, cmd)
+          if host_run
+            @logger.info("Running command (#{desc}) on host")
+            run_cmd(host_run)
+          end
```

Comment:
run_post_start_commands is called from HCKClient#configure which is called for each client. So, the run_post_start_commands will be called for each client too, but the host is one and we don't need to execute the host command twice.